src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1093,10 +1094,11 @@ private String getContainerCommand() {
         command.append(" -v " + serverDirectory + "/dropins:/config/dropins");
 
         // mount the loose application resources in the container
-        command.append(" -v "+projectDirectory.getAbsolutePath()+":"+DEVMODE_DIR_NAME);
+        command.append(" -v " + projectDirectory.getAbsolutePath() + ":" + DEVMODE_DIR_NAME);
+        command.append(" -v " + serverDirectory.getAbsolutePath() + DEVMODE_CONFIG_XML + ":/config" + DEVMODE_CONFIG_XML);
```

Comment:
1. Does this need a slash between `serverDirectory.getAbsolutePath() + DEVMODE_CONFIG_XML`?
2. Does this need a slash between `":/config" + DEVMODE_CONFIG_XML`?